recipes/lunarcalendar/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "LunarCalendar" %}
+{% set version = "0.0.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 681142f22fc353c3abca4b25699e3d1aa7083ad1c268dce36ba297eca04bed5a
+
+build:
+  number: 0
```

```suggestion
  number: 0
  noarch: python
```

The build failed before because you're missing `entry_points`:
* https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points
* https://github.com/wolfhong/LunarCalendar/blob/master/setup.py#L45

---

Thanks, that fixed it! (https://github.com/conda-forge/staged-recipes/pull/10885/commits/c42404a69613d8e81d190c94edc7dddf889817fb)